index.html

```diff
@@ -1,106 +1,114 @@
 ---
 layout: default
 ---
-<div class="main-content">
-    <div class="template-home">
-        <a id="mlh-trust-badge" href="https://mlh.io/seasons/na-2017/events?utm_source=na-2017&utm_medium=TrustBadge&utm_campaign=na-2017&utm_content=gray" target="_blank"><img src="{{ "img/mlh-trust-badge.svg" | prepend: site.baseurl }}" alt="Major League Hacking 2017 Hackathon Season" style="width:100%"></a>
+<div class="template-home">
+    <a id="mlh-trust-badge" href="https://mlh.io/seasons/na-2017/events?utm_source=na-2017&utm_medium=TrustBadge&utm_campaign=na-2017&utm_content=gray" target="_blank"><img src="{{ "img/mlh-trust-badge.svg" | prepend: site.baseurl }}" alt="Major League Hacking 2017 Hackathon Season" style="width:100%"></a>
```

Comment:
The trust badge isn't clickable since it's showing up under the `<section>`. Adding a z-index fixes it. Remember to update this to the 2018 badge/link.

Reply:
I thought we're still in the 2017 season though?

Reply:
I think the seasons are based on the school year since the 2018 page has fall events in it